srcpkgs/pam_zfscrypt/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'pam_zfscrypt'
+pkgname=pam_zfscrypt
+version=0.2.0
+revision=1
+wrksrc="zfscrypt-${version}"
+build_style=gnu-makefile
+make_use_env=yes
+hostmakedepends="zfs-devel"
+makedepends="pam-devel zfs-devel"
+make_dirs="/usr/lib/security 0755 root root"
```

Comment:
This looks wrong, the package contains files in that directory there is no need to create it.